### PR TITLE
Feat/reminder

### PR DESCRIPTION
**Pull request by Xazin**
### Feature Preview

---

#### PR Checklist

- [x] My code adheres to the [AppFlowy Style Guide](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions/submitting-code/style-guides)
- [ ] I've listed at least one issue that this PR fixes in the description above.
- [x] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [ ] All existing tests are passing.

### CODE REVIEW

**Comment by annieappflowy** [comment]
Can this feature get included in v0.3.3?
Next time, we should do our best to break a large-sized PR into smaller pieces or steps. This will result in quicker iterations and more concise feedback loops. What we must avoid is building something beyond the MVP that isn't desired by the customers.

**Comment by Xazin** [comment]
Waiting for https://github.com/AppFlowy-IO/AppFlowy-Collab/pull/103 to introduce read behavior on reminders.

File: frontend/rust-lib/flowy-date/src/event_handler.rs [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+#[tracing::instrument(level = "debug", skip_all, err)]
+pub(crate) async fn query_date_handler(
+  data: AFPluginData<DateQueryPB>,
+) -> DataResult<DateResultPB, FlowyError> {
+  let query: String = data.into_inner().query;
+  let date = DateParser::parse(&query);
+
+  let formatted: String;
+  if date.is_some() {
+    let naive_date = date.unwrap();
```
Comment (appflowy) [inline frontend/rust-lib/flowy-date/src/event_handler.rs:18]: It seems like you are using lot of unwrap. Please don't use unwrap unless it's safe to unwrap

**Comment by LucasXu0** [comment]
@Xazin I found some new issues.

Screenshot 2023-09-26 at 16 16 06
Screenshot 2023-09-26 at 16 24 24

1. it always shows 1/1/1970 for the first time.
2. the text color when hovering should be the same as the text color of the slash menu item.

--- 
Screenshot 2023-09-26 at 16 21 15

the reminder menu always shows at the beginning of the line instead of at the @ character's position.

---
I added a reminder for today, but it doesn't show in the notification center.
Screenshot 2023-09-26 at 16 28 53

**Pull request merged**